providers/cloudflare/dns.go

```diff
@@ -102,30 +107,23 @@ func (g *DNSGenerator) InitResources() error {
 			g.Resources = append(g.Resources, tmpRes...)
 		}
 	}
-
 	g.PopulateIgnoreKeys()
 	return nil
 
 }
 
 func (g *DNSGenerator) PostConvertHook() error {
-	for i, resourceRecord := range g.Resources {
-		if resourceRecord.InstanceInfo.Type == "cloudflare_zone" {
-			continue
-		}
-
-		item := resourceRecord.Item
-		zoneID := item["zone_id"].(string)
-		for _, resourceZone := range g.Resources {
-			if resourceZone.InstanceInfo.Type != "cloudflare_zone" {
-				continue
-			}
-			if zoneID == resourceZone.InstanceState.ID {
-				if resourceRecord.InstanceInfo.Type == "cloudflare_record" {
-					g.Resources[i].Item["domain"] = "${cloudflare_zone." + resourceZone.ResourceName + ".zone}"
```

Comment:
What about connect zone to record? It's not working?

Reply:
It works but this resource work well with zone (domain) name, so instead of using connection to 'zone', I use domain directly for easier to read. How do you thing about this ?